test/unit/command/test_idempotence.py

```diff
@@ -50,23 +50,25 @@ def test_execute_does_not_raise_on_converge_error(
 
 def test_execute_does_not_raise_on_idempotence_failure(
         patched_converge, patched_logger_error, molecule_instance):
-    patched_converge.return_value = None, 'changed=2'
+    output = 'check-command-01: ok=2    changed=1    unreachable=0    failed=0'
+    patched_converge.return_value = None, output
 
     i = idempotence.Idempotence({}, {}, molecule_instance)
     result = i.execute(exit=False)
 
-    msg = 'Idempotence test failed.'
+    msg = ''
```

Comment:
Oh, I see..... So really the problem is [trying to assert 2 calls to the same mock], and we see the last one (https://github.com/rgreinho/molecule/blob/8e801284089d4545147cac66296e05b04b3c5add/molecule/command/idempotence.py#L59-L61).  

I think you could do something like [this](https://github.com/metacloud/molecule/blob/ab46e9e7fc5808ebce57a9f1d5db4271e12f981d/test/unit/command/test_converge.py#L159-L165).